wiki/Community/Project_Loved/en.md

```diff
@@ -77,6 +75,7 @@ For any questions regarding what goes in each category, contact the [osu! Loved
 
 **Guidelines may be ignored under exceptional circumstances.** These exceptional circumstances can be justified by an explanation as of why the guideline has been ignored and why not ignoring it will interfere with the overall quality of the creation.
 
+- **The beatmap must be submitted at least 1 year before Loved nomination, except for beatmaps that break ranking criteria due to their concept.** Submitting a beatmap solely for Loved is discouraged when Ranked might be a better option. Exceptions include sliderator usage, Aspire maps, storyboard gameplay gimmicks, etc.
```

Comment:
why is this being moved to guidelines? it's a hard rule that doesn't have much ambiguity.

if you're proposing changes to rules and guidelines, they should be passed on and discussed with the Loved team.

best to move this back to rules for now.

Reply:
Talked about this in the osu dev Discord and this is what was huu told me, so I moved it there

![Loved Guideline](https://github.com/user-attachments/assets/88a188fd-70de-45f3-be9b-1eb617984385)

Reply:
in that case, it's recommended to use slightly less imperative wording for guidelines
```suggestion
- **The beatmap should be submitted at least 1 year before Loved nomination, except for beatmaps that break ranking criteria due to their concept.** Submitting a beatmap solely for Loved is discouraged when Ranked might be a better option. Exceptions include sliderator usage, Aspire maps, storyboard gameplay gimmicks, etc.
```